src/framework/mpas_attlist.F

```diff
@@ -565,7 +819,6 @@ subroutine mpas_remove_att(attList, attName, ierr)!{{{
                deallocate(cursor % attValueIntA)
             end if
             deallocate(cursor)
-            
```

Comment:
Since we aren't otherwise modifying the `mpas_remove_att` routine, I'd suggest just keeping this blank line as it was.